app/javascript/components/application_form/preferences/utils.js

```diff
@@ -0,0 +1,4 @@
+
+export const FIELD_NAME = 'shortFormPreferences'
```

Comment:
I'm not seeing the value of extracting this as a constant since it's only used in `PreferenceForm` right now.

Reply:
It's being used here https://github.com/Exygy/sf-dahlia-lap/pull/32/files#diff-09439580e565a0ba8421f21a86528a21R10